### DISCUSSION TITLE
Guest accounts

### DISCUSSION THREAD

**Discussion by gfreccia** [ltsp/ltsp#241] [Q&A]
Hi Alkis,

I've followed this _https://github.com/ltsp/community/wiki/Guest-session_, and now I've my guest accounts (from my template) working properly.
But it doesn't work this "This deletes the /home folder of the user logging in and creates a new one", so, when the guest user changes something (e.g. desktop background) it remains for the next sessions.

Giacomo

**Comment by alkisg** [comment]
Hi Giacomo,

I didn't write this wiki page, so there's no need to ask me specifically; ask either the page author or better yet the community in general. :)

**Comment by gfreccia** [comment]
Hi Dur0k!

Any hint here?

**Comment by alkisg** [comment]
Hi Giacomo,

keep in mind that this is a community site, not my personal site, so it's best to file issues without saying "Alkis" in them. Otherwise people won't answer because they'll think that this issue is only for me. And I won't answer because I didn't write that wiki page and I don't use guest setups, so you won't get any answers at all!

Cheers!

**Comment by gfreccia** [comment]
Hi All,

at the end I've decided to change the approach.
I use one script to create the guest users and another script to delete it, and I run both during night.
So every day I've fresh guests accounts.
Here my scripts.

_#createguests.sh
base="guest"
pass="guest"
mkdir /home/nfs
for ip in {20..34}; do
user="${base}${ip}"
adduser $user --disabled-password --gecos ""
echo -e "$pass\n$pass\n" | sudo passwd $user
mv "/home/${user}" /home/nfs/
ln -s "nfs/${user}" "/home/${user}"
done

#deleteguests.sh
base="guest"
pass="guest"
mkdir /home/nfs
for ip in {20..34}; do
user="${base}${ip}"
deluser --remove-home $user
done
rsync -a --delete empty_dir/ /home/nfs/
rm -f /home/$user_

But I've still two problems.

I've this on ltsp.conf to have a shared folder between users and the server.
FSTAB_SHARE="172.22.18.250:/share /home/share nfs defaults,nolock 0 0"
I've this on exports of the server:
/share 172.22.18.0/24(rw,sync,no_root_squash)
I see this fstab on the client; the folder is created but it's not shared!
172.22.18.250:/share /home/share nfs defaults,nolock 0 0

Then I wish to power off the client and I've put this on ltsp.conf, but no success.
CRONTAB_x="30 22 * * * poweroff"
In cron.d/ltsp on the client I see this:
30 22 * * * poweroff

Thank you!
